packages/Manager/src/main/kotlin/nz/govt/eop/plan_limits/Controller.kt

```diff
@@ -96,8 +96,7 @@ class Controller(val context: DSLContext, val queries: Queries, val manifest: Ma
       throw ResponseStatusException(
           HttpStatus.BAD_REQUEST, "The parameter \"to\" must be after \"from\"")
     }
-    val difference = DAYS.between(from, to)
-    if (difference < 1 || difference >= 366) {
+    if (YEARS.between(from, to) > 0) {
       throw ResponseStatusException(
           HttpStatus.BAD_REQUEST,
           "The duration between \"from\" and \"to\" should be more than one day and at most one year")
```

Comment:
This was the other bug, that throwing an exception here doesn't return a useful response. This can be done a few ways, but even `ResponseEntity.badRequest() ...` would be a simple fix